Bugzilla/Config/Auth.pm

```diff
@@ -145,6 +145,18 @@ sub get_param_list {
             type    => 't',
             default => '',
         },
+
+        {
+            name => 'mfa_group',
+            type => 't',
+            default => '',
+        },
```

Comment:
this should be a group select, not a text field (as per most of the fields in Bugzilla::Config::GroupSecurity).